source/aggregation-pipeline-builder.txt

```diff
@@ -112,35 +112,31 @@ your chosen view mode:
       .. procedure:: 
          :style: normal
 
-         .. step:: In the aggregation pipeline pane, ensure the 
+         .. step:: Select the :guilabel:`Stage` view.
+            
+            In the aggregation pipeline pane, ensure the 
             :guilabel:`{} Stages` toggle switch is selected.
 
             .. figure:: /images/compass/agg-builder-stageview-toggle-on.png
                :alt: Toggle on stage view mode 
 
-         .. step:: At the bottom of the aggregation pipeline pane, select the 
-            :guilabel:`+ Add Stage` button to open a new aggregation stage 
-            modal.
+         .. step:: Add an aggregation stage. 
+            
+            At the bottom of the aggregation pipeline pane, click the 
+            :guilabel:`+ Add Stage` button.
+            
+         .. step:: Select an aggregation pipeline stage.
             
-         .. step:: At the lower-left corner of the aggregation pipeline pane, 
+            On the upper-left corner of the aggregation stage card, 
             click the :guilabel:`Select...` dropdown and select the 
```

Comment:
[FYI] When there's an ellipsis in the UI, we don't add it to the docs content. [See here for more info](https://learn.microsoft.com/en-us/style-guide/punctuation/ellipses). Also, usually we would call this a _menu_ rather than a _dropdown_.

[suggestion] "On the upper-left corner of the aggregation stage card, use the `Select` menu to choose the first [aggregation pipeline stage](https://www.mongodb.com/docs/manual/reference/operator/aggregation-pipeline/)."

Reply:
Would it be too verbose to say "use the `Select` drop-down menu"? I think this is mainly because "menu" by itself can seem a bit vague in terms of UI elements.

Reply:
I needed to do some research here! A couple maybe-useful links I found while thinking about this: 
- [Describing UI elements](https://learn.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui)
- [dropdown](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/d/dropdown)

Based on the advice here, I wonder if it makes sense to just say, "... click **Select** to choose..."? Your call!